src/connector/src/parser/mod.rs

```diff
@@ -742,9 +776,9 @@ impl ByteStreamSourceParserImpl {
         let protocol = &parser_config.specific.protocol_config;
         let encode = &parser_config.specific.encoding_config;
         match (protocol, encode) {
-            (ProtocolProperties::Plain, EncodingProperties::Json(_)) => {
-                JsonParser::new(parser_config.specific, rw_columns, source_ctx).map(Self::Json)
-            }
+            // (ProtocolProperties::Plain, EncodingProperties::Json(_)) => {
+            //     JsonParser::new(parser_config.specific, rw_columns, source_ctx).map(Self::Json)
+            // }
```

Comment:
What's wrong? 👀

Reply:
we are migrating `plain json` case to `plain parser`. The only behavior difference is unnesting array, which we decided to deprecate

Reply:
I see. Then let's delete this.